packages/commonwealth/client/scripts/controllers/app/webWallets/cosmos_evm_metamask_web_wallet.ts

```diff
@@ -79,17 +78,20 @@ class CosmosEvmWebWalletController implements IWebWallet<string> {
     return this._chainId;
   }
 
-  public async signCanvasMessage(
-    account: Account,
-    canvasSessionPayload: SessionPayload,
-  ): Promise<string> {
-    const canvas = await import('@canvas-js/interfaces');
-    const signature = await this._web3.eth.personal.sign(
-      canvas.serializeSessionPayload(canvasSessionPayload),
-      this._ethAccounts[0],
-      '',
-    );
-    return signature;
+  public getSessionSigner() {
+    return new CosmosSignerCW({
+      bech32Prefix: app.chain?.meta.bech32Prefix || 'inj',
```

Comment:
Why default to injective here?

Reply:
That's what the corresponding value defaulted to before so I decided not to touch it.

Reply:
Seems reasonable!